yt_idefix/fields.py

```diff
@@ -15,6 +17,19 @@ class IdefixVtkFieldInfo(FieldInfoContainer):
     )
 
     def setup_fluid_fields(self):
+        unit_system = self.ds.unit_system
+
+        def _temperature(field, data):
+            mu = getattr(data.ds, "mu", compute_mu(data.ds.default_species_fields))
+            return mu * mh * data["gas", "pressure"] / data["gas", "density"] / kboltz
+
+        self.add_field(
+            name=("gas", "temperature"),
+            function=_temperature,
+            sampling_type="cell",
+            units=unit_system["temperature"],
+        )
```

Comment:
This goes beyond the initial intention for this PR, and I'm not sure this definition is correct for Idefix data, so I'd prefer if we could make this definition Pluto-only for now, and preferably in a different PR.

Reply:
OK. I'll retrieve this implement and create a new PlutoFieldInfo in the future when I prepare to.

Reply:
thanks !